src/app/components/error/ErrorBoundary.js

```diff
@@ -64,9 +64,21 @@ const errbitNotifier = ({
 
 const notifySentry = (
   error,
+  component,
+  callIntercom,
 ) => {
   if (config.sentryDsn) {
-    Sentry.captureException(error);
+    Sentry.captureException(error, {
+      contexts: {
+        component: {
+          name: component,
+        },
+      },
+    });
+  }
+  // even if sentry isn't configured we should still call Intercom
+  if (callIntercom) {
+    callIntercom({ url: window.location.href });
```

Comment:
Currently, the url passed to `callIntercom` is the url of the errbit issue, not the check url. This helps us match the user's complaint with the error.

Reply:
We get it from the response to the errbit api request. Does Sentry return it as well?

Reply:
Ohhh I misunderstood. I'll look into it.

Reply:
Check out [b17a696](https://github.com/meedan/check-web/pull/1613/commits/b17a69683e3b57d68c1db811799a5baea110d478) for the update.

Reply:
Nice! Thanks